Descent3/Controls.cpp

```diff
@@ -805,11 +799,6 @@ void DoMovement(game_controls *controls) {
     controls->afterburn_thrust = LIMIT_FORWARD;
   if (controls->afterburn_thrust < -LIMIT_FORWARD)
     controls->afterburn_thrust = -LIMIT_FORWARD;
-
-  mprintf_at(1, 5, 30, "ch:%.2f ", controls->heading_thrust);
```

Comment:
no replacement?

Reply:
`mprintf_at()` is no-op, it can be safely removed. Will remove after ddebug cleanup eventually.